src/components/TodoErrors.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import { ErrorType } from '../types/ErrorType';
+
+type Props = {
+  error: ErrorType | null;
+};
+
+export const TodoErrors: React.FC<Props> = ({ error }) => {
+  const [visible, setVisible] = useState(true);
```

Comment:
Rename all boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [iVisible, setIsVisible] = useState(true);
```